src/main/scala/io/iohk/ethereum/consensus/ethash/difficulty/ConfiguredDifficultyCalculator.scala

```diff
@@ -0,0 +1,34 @@
+package io.iohk.ethereum.consensus.ethash.difficulty
+
+import io.iohk.ethereum.consensus.difficulty.DifficultyCalculator
+import io.iohk.ethereum.domain.BlockHeader
+
+class ConfiguredDifficultyCalculator(powTargetTime: Long) extends DifficultyCalculator {
+
+  import EthashDifficultyCalculator._
+
+  /**
+    * The LowerBoundExpectedRatio (l for abbreviation below) divides the timestamp diff into ranges:
+    *   [0, l)   => c = 1, difficulty increases
+    *   [l, 2*l) => c = 0. difficulty stays the same
+    *   ...
+    *   [l*i, l*(i+1) ) => c = 1-i, difficulty decreases
+    *
+    *  example:
+    *  powTargetTime := 45 seconds
+    *  l := 30 seconds
+    *   [0, 0.5 min)         => difficulty increases
+    *   [0.5 min, 1 min)   => difficulty stays the same (the average should be powTargetTime)
+    *   [1 min, +infinity) => difficulty decreases
+    */
+  private val LowerBoundExpectedRatio: Long = (powTargetTime / 1.5).toLong
```

Comment:
A nazi comment: capitalised names are for constants and this is not a constant if it depends on the class parameter